packages/next/src/server/lib/patch-fetch.ts

```diff
@@ -151,12 +151,11 @@ export function createPatchedFetcher(
   originFetch: Fetcher,
   { workAsyncStorage, workUnitAsyncStorage }: PatchableModule
 ): PatchedFetcher {
-  // Create the patched fetch function. We don't set the type here, as it's
-  // verified as the return value of this function.
-  const patched = async (
+  // Create the patched fetch function.
+  const patched = async function fetch(
     input: RequestInfo | URL,
```

Comment:
It should set a name descriptor like we do in React with console patching so that it isn't subject to mangling: https://github.com/facebook/react/blob/b44a99bf58d69d52b5288d9eadcc6d226d705e11/packages/react-server/src/ReactFlightServer.js#L296-L303

Reply:
I only thought about stacks in the dev overlay, because in the terminal they can't be unhidden. But you're right, why not define the name as a property in case that ever changes. Will fix...